crates/bevy_pbr/src/render/mesh.rs

```diff
@@ -289,6 +387,22 @@ impl FromWorld for MeshPipeline {
             }],
             label: Some("mesh_layout"),
         });
+
+        let skinned_mesh_layout =
+            render_device.create_bind_group_layout(&BindGroupLayoutDescriptor {
+                entries: &[BindGroupLayoutEntry {
+                    binding: 0,
+                    visibility: ShaderStages::VERTEX,
+                    ty: BindingType::Buffer {
+                        ty: BufferBindingType::Uniform,
+                        has_dynamic_offset: true,
+                        min_binding_size: BufferSize::new(JOINT_BUFFER_SIZE as u64),
+                    },
+                    count: None,
+                }],
+                label: Some("mesh_layout"),
```

Comment:
```suggestion
                label: Some("joint_buffer_layout"),
```
Nit: distinct name for debugging.